src/server.cpp

```diff
@@ -405,7 +406,7 @@ CServer::CServer ( const int          iNewMaxNumChan,
     if ( !strRecordingDirName.isEmpty() )
     {
         bRecorderInitialised = JamRecorder.Init ( this, iServerFrameSizeSamples );
-        bEnableRecording     = bRecorderInitialised;
+        SetEnableRecording ( bRecorderInitialised );
```

Comment:
Why call the function? If bRecorderInitialised is false, the function body in SetEnableRecording is not entered. If it is true, the special implementation in the if-condition is not reached. So there is no difference in just applying the new value to the boolean variable. Do you have something else in mind why you changed this? Do you want to put new functionality in the SetEnableRecording function?

Reply:
Ah...  Ahh.....  AAHH!!! Yes... very good point.  Oops.

Reply:
...except if it's true, I want the message, and I'd rather have the code for the message only once...

Reply:
Ok, makes sense. Thanks for the explaination.